### PR TITLE
Mm task admin app sprint 2

### PR DESCRIPTION
**Pull request by dejetem**
https://user-images.githubusercontent.com/52535603/233185236-bff73a66-9ec2-4207-aebf-1f6e7182a2f6.mp4

https://user-images.githubusercontent.com/52535603/233652150-13fdc3c2-9727-420c-8c65-92b61ba73da8.mp4

### CODE REVIEW

**Review by emmsdan — CHANGES_REQUESTED** [review]
- Please, add functionality to the sidebar to enable users to select different tasks to display on the main task area
- when a user selects a task, it should show on the sidebar
- also please make this responsive

**Comment by dejetem** [comment]
> * Please, add functionality to the sidebar to enable users to select different tasks to display on the main task area
> * when a user selects a task, it should show on the sidebar
> * also please make this responsive

Okay

**Comment by dejetem** [comment]
> * Please, add functionality to the sidebar to enable users to select different tasks to display on the main task area
> * when a user selects a task, it should show on the sidebar
> * also please make this responsive

but their is no mobile design on the figma

File: frontend/mms-Admin/public/assets/images/Mentor.svg
Comment (shegs-devv) [inline frontend/mms-Admin/public/assets/images/Mentor.svg:1]: These icons have been added already

Comment (dejetem) [reply frontend/mms-Admin/public/assets/images/Mentor.svg:1]: Okay

**Review by Shegsdev — APPROVED** [review]
Very well

**Review by feisal-Codes — APPROVED** [review]

**Review by emmsdan — APPROVED** [review]

**Pull request merged**